source/config/Config.cpp

```diff
@@ -113,7 +113,19 @@ bool PlainConfig::LoadFromJson(const Crt::JsonView &json)
     {
         if (!json.GetString(jsonKey).empty())
         {
-            rootCa = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
+            basic_string<char> path = FileUtils::ExtractExpandedPath(json.GetString(jsonKey).c_str());
+            if (!FileUtils::FileExists(path))
+            {
+                LOGM_ERROR(
```

Comment:
If we aren't going to fail device client due to an invalid path, then level should be `LOGM_WARN` instead of `LOGM_ERROR`.

Reply:
You also need similar logic in `PlainConfig::LoadFromCliArgs`. https://github.com/awslabs/aws-iot-device-client/blob/c53400022f87ac0cb7fe8b8f3bb18e4f5198ca26/source/config/Config.cpp#L225